nilearn/plotting/img_plotting.py

```diff
@@ -901,7 +900,7 @@ def plot_stat_map(stat_map_img, bg_img=MNI152TEMPLATE, cut_coords=None,
     stat_map_img = _utils.check_niimg_3d(stat_map_img, dtype='auto')
 
     cbar_vmin, cbar_vmax, vmin, vmax = _get_colorbar_and_data_ranges(
-        stat_map_img,
+        stat_map_img.get_data(),
```

Comment:
I was going to open an issue about that. The original code used `get_data()` so I kept it to minimize the impact of the PR.